common/common.go

```diff
@@ -0,0 +1,58 @@
+package common
+
+import (
+	"math/big"
+	"os"
+	"strconv"
+
+	"github.com/flashbots/go-boost-utils/types"
+)
+
+const (
+	GenesisTimeMainnet = 1606824023
+	SlotTimeSecMainnet = 12
+)
```

Comment:
I would have this grouped under types.go with all the network constants or separate into a new const file

Reply:
maybe types.go should go into the common package